libs/expo/betterangels/src/lib/screens/Clients/index.tsx

```diff
@@ -187,7 +187,7 @@ export default function Clients({ Logo }: { Logo: ElementType }) {
             setClients({});
           }}
         />
-        {search && !loading && sections.length < 1 && (
+        {search && !loading && sections.length < 1 ? (
```

Comment:
I kind of have issues with long and nested `ternary` expressions. I won't block on this, but it seems pretty difficult to read especially as the ternary has its own nested ternarys.

Maybe try something like this.

1. define `const` for `hasResults` (or `visibleSearchResults` or ....)


```javascript
const hasResults = search && !loading && !!sections.length; // or however we define it
```

2.  Could create separate component for no-results: `< NoSearchResultsComponent />`? Could keep it in same folder and specific to this screen for now, though eventually can have a generic "no results found" display.

3.  Render based on `hasResults`

```javascript

{hasResults && (
    <SearchResultsComponent />
)}

{!hasResults && (
    <NoSearchResultsComponent />
)}
```

Reply:
replaced the dirty ternary. added tickets to [create](https://betterangels.atlassian.net/browse/DEV-942) and [implement](https://betterangels.atlassian.net/browse/DEV-943) reusable "no results" component